.github/workflows/pull-request.yml

```diff
@@ -38,8 +38,8 @@ jobs:
       - name: Install dependencies
         run: pnpm install --frozen-lockfile
 
-      - name: Run prettier
-        run: pnpm run lint
+      - name: Run Biome
+        run: pnpm biome ci && pnpm biome ci --reporter summary
```

Comment:
Why does the command get run twice here?

Reply:
The first one lists all results of the check, the second one is just a summary.
The summary can be removed if not needed. I thought it was a nice feature